contracts/interfaces/IDeployVerifier.sol

```diff
@@ -16,8 +16,10 @@ interface IDeployVerifier {
      *              Note that in most cases the verifier shouldn't try use it at all. It is always checked
      *              by the forwarder immediately after verifyRelayedCall returns.
      */
-   function verifyRelayedCall(
+    function verifyRelayedCall(
         EnvelopingTypes.DeployRequest calldata relayRequest,
         bytes calldata signature
     ) external returns (bytes memory context);
+
+    event AllowedToken(address indexed tokenAddress);
```

Comment:
There is no need to add events to determine which tokens are accepted, the relay server can query the state of the contract directly and that is an off-chain call without any cost. This scales better than the use of events. For simplicity, the verifiers can have a list of accepted tokens (additionally to the current map) so parsing the state is easier.